horace_core/sqw/@axes_block/axes_block.m

```diff
@@ -396,16 +390,23 @@
             %           -- if provided, returns grid used to define density,
             %              namely with points located on the grid cell edges +
             %              edges of integrated  dimensions.
-            % '-density_integr'
+            % '-bin_center'
             %           -- if provided, returns grid used for integration
-            %              by summation in centerpoints, namely, points
-            %              are in the center of cells and integration
+            %              by summation in centre-points, namely, points
+            %              are in the centre of cells and integration
```

Comment:
Inconsistent spelling of `centre`, pick one or it could get confusing if one reads the docs, and assumed the parameter was the same.

Reply:
done